src/java/org/apache/cassandra/net/MessagingService.java

```diff
@@ -254,6 +258,8 @@ public long getTimeout()
                 return DatabaseDescriptor.getRangeRpcTimeout();
             }
         },
+        PING(),
```

Comment:
Left a similar comment on ticket, but this should be added at the end:
*** 
// remember to add new verbs at the end, since we serialize by ordinal
        UNUSED_1,
        UNUSED_2,
        UNUSED_3,
        UNUSED_4,
        UNUSED_5,

Reply:
See discussion on ticket, but I'll clean up the enum and comments